### DISCUSSION TITLE
TC Pricing

### DISCUSSION THREAD

**Discussion by ynot01** [yogstation13/Yogstation#10228] [Discussions]
# https://wiki.yogstation.net/wiki/Syndicate_Items
### What do you think should be increased in price and why?

**Comment by TheLinuxOS** [comment]
EMP Implant kit increase from 2 TC ----> 4 TC. Currently, it is the exact same price and overall more useful than the EMP flashlight. It is unbalanced in terms of cost if you just look at the number of uses you can get out of it and the number of scenarios you can use it in for just 2 TC.

Binary Translator Key decrease from 5 TC ----> 4 TC. This isn't as big of a change in my mind at just makes sense, it is an item that gives you only access to one radio and is very niche in its use. This reduction would also make it the same price as the hacked AI law board and there is little that can be said to convince me that being able to see the binary channel is more important than subverting an AI with a law 0. It would also allow you to combo an AI detector with the binary translator for the previous price of the binary translator.

**Reply by ynot01** [reply]
EMP kit is more destructive and uncontrolled, generally nets more attention while emp flashlight is silent and recharges

agree to the binary key change, it's way underused for such a unique gimmicky item

**Comment by Moja10** [comment]
Syndicate minibombs should be 4 TC, they were 4 TC prebase and rebase brought them to 6 TC. I never see anyone buy minibombs unless they are new to the game or on discount. They basically have one real use, which is running in and dumping one on cloning, or your AFK target. In terms of breaking into places and destroying critical machinery, C4 does the same price and for the price of 1 3/4 minibombs, you can get 10 C4, and C4 is also granular since you can buy singular ones. In terms of big explosions, the syndicate bomb is way superior because it will actually incapacitate a department or hit your target through walls, or cause enough mayhem so you can have a distraction while people recover/rebuild after the explosion. Minibombs are just... at 6 TC they literally compete with good items like emag, are completely outclassed by other explosives, and are shit as a weapon. No one would buy 2 minibombs for attack or defense compared to a revolver or sketchkin. If it was totally up to me I would legitimately make them 3 TC but I think 4 TC would work too.
